torchao/quantization/_eval.py

```diff
@@ -0,0 +1,228 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Can we put this in a higher level namespace? Something like torchao._eval? or torchao.model._eval or torchao.util